System.IO.FileSystem/File.cs

```diff
@@ -9,23 +9,63 @@
 namespace System.IO
 {
     /// <summary>
-    /// Class for creating FileStream objects, and some basic file management
-    /// routines such as Delete, etc.
+    /// Provides static methods for the creation, copying, deletion, moving, and opening of a single file, and aids in the creation of <see cref="FileStream"/> objects.
     /// </summary>
     public static class File
     {
         private const int ChunkSize = 2048;
         private static readonly byte[] EmptyBytes = new byte[0];
 
+        /// <summary>
+        /// Creates a <see cref="StreamWriter"/> that appends UTF-8 encoded text to an existing file, or to a new file if the specified file does not exist.
+        /// </summary>
+        /// <param name="path">The path to the file to append to.</param>
+        /// <returns>A stream writer that appends UTF-8 encoded text to the specified file or to a new file.</returns>
+        public static StreamWriter AppendText(string path)

```

Comment:
I'm finding this method name to be confusing, specially when considering the next method which is `AppendAllText`. I think this one can be more explicit like `CreateWriter` while the other method can be called `AppendText`.

Reply:
Both are comming from .NET API...
Please take a look at [`AppendText`](https://learn.microsoft.com/dotnet/api/system.io.file.appendalltext?view=netframework-4.8.1&WT.mc_id=DT-MVP-5004179) and [`AppendText`](https://learn.microsoft.com/dotnet/api/system.io.file.appendtext?view=netframework-4.8.1&WT.mc_id=DT-MVP-5004179)

Reply:
Ok, I see. Thanks for that.